client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -119,8 +119,19 @@ class DomainRegistrationSuggestion extends React.Component {
 	renderDomain() {
 		const {
 			suggestion: { domain_name: domain },
+			translate
 		} = this.props;
-		return <h3 className="domain-registration-suggestion__title">{ domain }</h3>;
+
+		let isAvailable = false;
+
+		//If we're on the Mapping or Transfer pages, add a note about availability
+		if ( includes( page.current, '/mapping' ) || includes( page.current, '/transfer' ) ) {
+			isAvailable = true;
+		}
+
+		const title = isAvailable ? translate( '%s is available!', { args: domain } ) : domain;
```

Comment:
Hi! I've found a possible matching string that has already been translated **12** times:
`translate( '%s is available.' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domains-transfer-standardization)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).